### PR TITLE
Members Collection and our team block

### LINKED ISSUE #602: @/codeforafrica Teams Block
Add teams Block to CFA CMS

image

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

Adds Members Collection and blockifies Our Team.

Fixes #602

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/codeforafrica/src/payload/collections/Members.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      },
+      localized: true,
+      required: true,
+    }),
+    socialLinks({
+      name: "connect",
+      label: "Social Media Links",
+      required: false,
+    }),
+    {
```
Comment (kelvinkipruto) [inline apps/codeforafrica/src/payload/collections/Members.js:70]: Perhaps Teams should be in its own collection as it is re-used for multiple people

**Review by kelvinkipruto — APPROVED** [review]

File: apps/codeforafrica/src/payload/collections/Members.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      },
+      type: "text",
+      required: true,
+    },
+    slug({ fieldToUse: "name" }),
+    {
+      name: "country",
+      type: "select",
+      options: allCountries,
+      label: { en: "country" },
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/collections/Members.js:53]:
```suggestion
      name: "country",
      label: { en: "Country" },
      type: "select",
      options: allCountries,
```

File: apps/codeforafrica/src/payload/blocks/OurTeam.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const OurTeam = {
+  slug: "our-team",
+  imageURL: "/images/cms/blocks/team.png",
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/blocks/OurTeam.js:3]: This is too big... use an image with one or two rows of members

File: apps/codeforafrica/src/payload/blocks/OurTeam.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  imageAltText: "Our Team",
+  labels: {
+    singular: {
+      en: "Our Team",
+    },
+    plural: {
+      en: "Our Team",
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/blocks/OurTeam.js:10]:
```suggestion
      en: "Team",
    },
    plural: {
      en: "Team",
```

File: apps/codeforafrica/src/payload/collections/Members.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    singular: {
+      en: "Member",
+    },
+    plural: {
+      en: "Members",
+    },
+  },
+  admin: {
+    useAsTitle: "name",
+  },
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/collections/Members.js:20]: Add default columns

File: apps/codeforafrica/src/payload/collections/Members.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+      label: "Social Media Links",
+      required: false,
+    }),
+    {
+      name: "team",
+      label: {
+        en: "Team",
+      },
+      type: "relationship",
+      relationTo: "teams",
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/collections/Members.js:71]: I think this needs to be required. Can't be a member of no team.

File: apps/codeforafrica/src/payload/collections/Teams.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import slug from "../fields/slug";
+
+const Teams = {
+  slug: "teams",
+  admin: {
+    useAsTitle: "name",
+  },
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/collections/Teams.js:7]: Add default columns

**Review by kilemensi — APPROVED** [review]

**Review by kilemensi — COMMENTED** [review]
👍🏽 ... Just had a quick look on the CMS side:

1. Lets arrange the page to resemble other CMS. Main part of the page should contain cover image, title, and content only. Side panel should contain slug, publish date, tags, excerpt (textarea and not richText), and author.
2. Aren't we keeping the old architecture of having just **Posts** collection with **Stories** and **Opportunities** using different tags?
3. Posts should also have SEO like pages

**Pull request merged**